js/aax.js

```diff
@@ -3238,6 +3238,82 @@ module.exports = class aax extends Exchange {
         };
     }
 
+    async fetchTransactionFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name aax#fetchTransactionFees
+         * @description fetch transaction fees
+         * @see https://www.aax.com/apidoc/index.html#get-currencies
+         * @param {[string]|undefined} codes list of unified currency codes
+         * @param {object} params extra parameters specific to the aax api endpoint
+         * @returns {object} a list of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure}
+         */
+        await this.loadMarkets ();
+        const request = {};
+        if (codes !== undefined) {
+            const codesLength = codes.length;
+            if (codesLength === 1) {
+                request['currency'] = this.safeValue (codes, 0);
+            }
+        }
+        const response = await this.publicGetCurrencies (this.extend (request, params));
+        return this.parseTransactionFees (response, codes);
+    }
+
+    parseTransactionFees (response, codes = undefined) {
+        const data = this.safeValue (response, 'data');
+        const result = {};
+        for (let i = 0; i < data.length; i++) {
+            const entry = data[i];
+            const currencyId = this.safeString (entry, 'currency');
+            const currency = this.safeCurrency (currencyId);
+            const code = this.safeString (currency, 'code');
+            if ((codes === undefined) || (this.inArray (code, codes))) {
+                const resultKeys = Object.keys (result);
+                if (!this.inArray (code, resultKeys)) {
+                    result[code] = {
+                        'withdraw': this.parseTransactionFee (entry),
+                        'deposit': {},
+                        'info': [],
+                    };
+                }
```

Comment:
Match the [format used here](https://github.com/ccxt/ccxt/pull/15582/files#r1015742616)